### PR TITLE
Add select dropdown

### PR DESCRIPTION
**Pull request by Juliiis**

### CODE REVIEW

**Comment by Juliiis** [comment]
Review please if I did ok this three points:
- added month selector
- added months map
- added empty state

File: src/Components/list.js [hunk 1]
```diff
@@ -14,13 +15,25 @@ function List() {
...
 
+    const handleMonthChange = (e) => {
+        setMonth(e.target.value)
+    }
+
     if (loading) {
         return 'Loading'
     }
 
+    const noFruitsInThisMonths = fruits.length === 0
```
Comment (PabloMDiez) [inline src/Components/list.js]: Maybe change `months` for `month`?

**Review by PabloMDiez — COMMENTED** [review]
Looks very good! A tiny grammar thing, though

File: src/Components/list.js (hunk 1 above)
Comment (Juliiis) [reply src/Components/list.js]: Ok, I'm going to change it now .. thanks!

**Review by PabloMDiez — APPROVED** [review]
Awesome ✌️

**Pull request merged**